inventories/providers/abb/ssc600_hypervisor_standalone_example.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2024, SFL (https://savoirfairelinux.com)
+# SPDX-License-Identifier: Apache-2.0
+
+---
+all:
+  children:
+    standalone_machine:
+      children:
+        hypervisors:
+          hosts:
+            rtei-demoHyp:
+              ansible_host: "192.168.216.73"
+              ip_addr: "{{ ansible_host }}"
+              network_interface: "enp0s20f0u7"
+              cluster_interface: "{{ network_interface }}"
+              gateway_addr: "192.168.216.1"
+              dns_server: "{{ gateway_addr }}"
+              ntp_servers: "{{ gateway_addr }}"
+              syslog_server_ip: "{{ gateway_addr }}"
+              apply_network_config: false
+              kernel_parameters_restart: false
+              ignored_bridges: []
+              nics_affinity:
+              - enp88s0: 3
+              ptp_interface: "enp0s20f0u3"
+              ovs_bridges:
+                - name: ovsbr
+                  ports:
+                    - name: VM1.0
+                      type: tap
+                    - name: ovsbr_ext
+                      type: system
+                      interface: "enp0s20f0u3"
+              upload_files: # This file is given with the ssc600 sw package
+                - { src: '../files/qemu.hook', dest: '/etc/modprobe.d/qemu-system-x86.conf', mode: "0744" }
```

Comment:
I think the path is wrong qemu.hook into /etc/modprobe.d/qemu-system-x86.conf. 

Furthermore, it will be more readable if you use the full develop form:

```yaml
    upload_files:
      - src: ../files/qemu.hook
        dest: /etc/libvirt/hooks/
        mode: "0744"

Reply:
Which path ? the src one ?

Reply:
dest

Reply:
this is the path specified in the doc

Reply:
My bad, I mixed two paths